internal/commands/manifest_create.go

```diff
@@ -0,0 +1,50 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/pkg/client"
+	"github.com/buildpacks/pack/pkg/logging"
+)
+
+// ManifestCreateFlags define flags provided to the ManifestCreate
+type ManifestCreateFlags struct {
+	format            string
+	insecure, publish bool
+}
+
+// ManifestCreate creates an image-index/image-list for a multi-arch image
+func ManifestCreate(logger logging.Logger, pack PackClient) *cobra.Command {
+	var flags ManifestCreateFlags
+
+	cmd := &cobra.Command{
+		Use:   "create <manifest-list> <manifest> [<manifest> ... ] [flags]",
+		Args:  cobra.MatchAll(cobra.MinimumNArgs(2), cobra.OnlyValidArgs),
+		Short: "Create a manifest list or image index.",
+		Example: `pack manifest create cnbs/sample-package:hello-multiarch-universe \
+		cnbs/sample-package:hello-universe \
+		cnbs/sample-package:hello-universe-windows`,
+		Long: `Generate manifest list for a multi-arch image which will be stored locally for manipulating images within index`,
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			return pack.CreateManifest(
+				cmd.Context(),
+				client.CreateManifestOptions{
+					IndexRepoName: args[0],
+					RepoNames:     args[1:],
+					Format:        flags.format,
+					Insecure:      flags.insecure,
+					Publish:       flags.publish,
+				},
+			)
+		}),
+	}
+
+	cmdFlags := cmd.Flags()
+
+	cmdFlags.StringVarP(&flags.format, "format", "f", "v2s2", "Format to save image index as ('OCI' or 'V2S2')")
```

Comment:
we took `v2s2` from [podman](https://docs.podman.io/en/latest/markdown/podman-manifest-push.1.html#format-f-format), in case people used that tool before. But I like maybe using `oci` and `docker`, we are actually parsing the user values to the corresponding `ggcr` media-type. I moved that logic to this file, let me push it and we can change it accordingly, I am happy to make it easier for end-users.